bitwarden_license/src/Commercial.Core/SecretsManager/Commands/Projects/CreateProjectCommand.cs

```diff
@@ -21,20 +23,44 @@ public class CreateProjectCommand : ICreateProjectCommand
         _projectRepository = projectRepository;
     }
 
-    public async Task<Project> CreateAsync(Project project, Guid userId)
+    public async Task<Project> CreateAsync(Project project, Guid id, ClientType clientType)
     {
+        if(clientType != ClientType.User && clientType != ClientType.ServiceAccount)
+        {
+            throw new NotFoundException();
+        }
+
         var createdProject = await _projectRepository.CreateAsync(project);
 
         var orgUser = await _organizationUserRepository.GetByOrganizationAsync(createdProject.OrganizationId,
```

Comment:
We should move fetching the user into the `if(clientType == ClientType.User)` block